packages/manager/src/features/CloudPulse/Alerts/CreateAlert/NotificationChannel/AddNotificationChannel.tsx

```diff
@@ -0,0 +1,201 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box } from '@linode/ui';
+import { Typography } from '@linode/ui';
+import { Autocomplete } from '@linode/ui';
```

Comment:
optimize this import statement as per eslint 

```suggestion
import { Box, Typography, Autocomplete } from '@linode/ui';
```

Reply:
@santoshp210-akamai , we should enable and use ESLint as formatter in plugin setting for these

Reply:
I have the ESLint formatter plugin, but for this it doesn't show any warning